tutorial4_timeseries/demo/main.js

```diff
@@ -164,27 +163,27 @@ function draw() {
             .attr("cy", height - margin.bottom)
             .remove()
         )
-    ).call(selection =>
-      selection
-        .transition() // initialize transition
-          .duration(1000) // duration 1000ms / 1s
-          .attr("cy", d => yScale(d.population)) // started from the bottom, now we're here
     )
-  
-    const line = svg
-      .selectAll("path.trend")
-      .data([filteredData])
-      .join(
-        enter => enter.append("path")
+    // the '.join()' function leaves us with the 'Enter' + 'Update' selections together.
+    // Now we just need move them to the right place
+    .transition() // initialize transition
+    .duration(1000) // duration 1000ms / 1s
+    .attr("cy", d => yScale(d.population)); // started from the bottom, now we're here
```

Comment:
Here we don't necessarily need to use `call` since we are not using this `selection` any further. I took it out to reduce code, but if you think that it is confusing we can bring it back. 
We technically can always use `call`s on transitions -- maybe that is just the best way to teach it? What do you think?

Reply:
yeah as I was compiling information about why and when we need it, I realized the same thing. I think since its like squares and rectangles, in my mind. It will always work with a call, but will sometimes screw you up if you don't have it. I'm partial to include it as a result, but i'm not super invested either way.